src/elements/reloc_section.rs

```diff
@@ -0,0 +1,244 @@
+use std::io::{Read, Write};
+
+use super::{CountedList, CountedListWriter, CountedWriter, Deserialize, Error, Serialize, VarInt32, VarUint32, VarUint7};
+
+const R_WEBASSEMBLY_FUNCTION_INDEX_LEB: u8 = 0;
```

Comment:
i would suggest naming

```rust
const FUNCTION_INDEX_LEB: u8 = 0;
const TABLE_INDEX_SLEB: u8 = 1;
```

to avoid the same prefix `R_WEBASSEMBLY_` everywhere

hiding all constants in inlined module with some common title shared by all constants as name is also fine

Reply:
I had copied the names from the spec. I agree that it isn't really necessary to keep the prefix since it makes sense in context, so I've dropped it.

Namespacing in a module isn't worth it, I think. The constants are only visible in the module anyway.